types/react-json-tree/index.d.ts

```diff
@@ -18,7 +19,7 @@ export interface JSONTreeProps extends Props<JSONTreeComponent> {
     sortObjectKeys?: Function | boolean;
     shouldExpandNode?: (keyPath: (string | number)[], data: [any] | {}, level: number) => boolean;
     getItemString?: (type: string, data: [any] | {}, itemType: string, itemString: string) => JSX.Element;
-    labelRenderer?: (raw: [string, string]) => JSX.Element;
+    labelRenderer?: (keyPath: string[], nodeType: string, expanded: boolean, expandable: boolean) => JSX.Element;
```

Comment:
are parameters beyond `keyPath` optional? i.e. `nodeType?: string`

Reply:
good pick up, they should be optional